databuilder/query_engines/query_model_convert_to_old.py

```diff
@@ -165,9 +165,12 @@ def convert_combine_as_set(node: new.AggregateByPatient.CombineAsSet):
 
 def convert_aggregation(node):
     function = AGGREGATE_MAP[node.__class__]
-    assert isinstance(node.source, new.SelectColumn)
-    source_table = convert_node(node.source.source)
-    input_column = column_name(node.source.name)
+    if not isinstance(node.source, new.SelectColumn):
```

Comment:
I understand that this isn't a change here, but why is the only `Series` source we accept a `SelectColumn`, not some other `Series` type?